pom.xml

```diff
@@ -134,7 +134,7 @@
     <hive.version.short>2.3</hive.version.short>
     <!-- note that this should be compatible with Kafka brokers version 0.10 and up -->
     <kafka.version>2.6.0</kafka.version>
-    <derby.version>10.12.1.1</derby.version>
```

Comment:
Could you add a comment, `After 10.15.1.3, the minimum required version is JDK9`, here?